pkg/iris/common/models/eventTypes.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+const (
+	EVENT_TYPE_STATE    string = "state"
```

Comment:
nitpicky but should conform to nomenclature

Reply:
Hey, what do you mean here? What would the expected nomenclature?

Reply:
i just meant it should follow the writing style of the other constants, it's no big deal 😅

Reply:
ah okay!